examples/apis/order/createtransaction/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/order"
+)
+
+func main() {
+	accessToken := "{{Access_token}}"
+	c, err := config.New(accessToken)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	client := order.NewClient(c)
+	orderRequest := order.Request{
+		Type:              "online",
+		ProcessingMode:    "manual",
+		TotalAmount:       "100.00",
+		ExternalReference: "ext_ref_1234",
+		Payer: order.PayerRequest{
+			Email: "{{EMAIL}}",
+		},
+	}
+
+	orderResource, err := client.Create(context.Background(), orderRequest)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	orderID := orderResource.ID
+	fmt.Println("Order created with success. Order IDL: ", orderID)
```

Comment:
[req] Foi uma letra L a mais na mensagem
```suggestion
	fmt.Println("Order created with success. Order ID: ", orderID)
```